libpod/image/image.go

```diff
@@ -338,10 +352,24 @@ func (i *Image) TagImage(tag string) error {
 	return i.imageruntime.store.SetNames(i.ID(), tags)
 }
 
+// UnTagImage removes a tag from the given image
+func (i *Image) UnTagImage(tag string) error {
```

Comment:
The capitalization here seems sort of wierd... `UntagImage` maybe?

Reply:
done